libunwind/src/Registers.hpp

```diff
@@ -1866,14 +1866,18 @@ class _LIBUNWIND_HIDDEN Registers_arm64 {
   void        setCapabilityRegister(int num, uintcap_t value);
 #else
   CAPABILITIES_NOT_SUPPORTED
-#endif
+#endif // __CHERI_PURE_CAPABILITY__
 
   uintptr_t  getSP() const          { return _registers.__sp; }
   void       setSP(uintptr_t value) { _registers.__sp = value; }
   uintptr_t  getIP() const          { return _registers.__pc; }
   void       setIP(uintptr_t value) { _registers.__pc = value; }
   uintptr_t  getFP() const          { return _registers.__fp; }
   void       setFP(uintptr_t value) { _registers.__fp = value; }
+#if defined(__CHERI_PURE_CAPABILITY__) && defined(_LIBUNWIND_CHERI_C18N_SUPPORT)
+  void       setTrustedStack(uintptr_t value) { _registers.__ecsp = value; }
+  uintptr_t  getTrustedStack() const { return _registers.__ecsp; }
```

Comment:
First get then set to match the functions above.

```suggestion
  uintptr_t  getTrustedStack() const { return _registers.__ecsp; }
  void       setTrustedStack(uintptr_t value) { _registers.__ecsp = value; }
```